doc/Hammer-Use/Hammer-APIs.rst

```diff
@@ -33,7 +33,58 @@ Pin constraints are specified using the Hammer IR key ``vlsi.inputs.pin``. PinAs
 
 Power Straps
 ------------
-Power strap constraints are specified using multiple Hammer IR keys in the ``par`` namespace. The currently supported API supports power strap generation by tracks, which auto-calculates power strap width, spacing, set-to-set distance, and offsets based on basic DRC rules specified in the technology Stackup object. The basic pieces of information needed are the desired track utilization per strap and overall power strap density. Different values can be specified on a layer-by-layer basis by appending ``_<layer name>`` to the end of the desired option.
+Power strap constraints are specified using multiple Hammer IR keys in the ``par`` namespace.You can find the keys in <tech>/defaults.yml under the tech plugin directory. An example from asap7 (https://github.com/ucb-bar/hammer/blob/master/hammer/technology/asap7/defaults.yml) is as follows:
+
+.. literalinclude:: ../../hammer/technology/asap7/defaults.yml
+   :language: yaml
+   :linenos:
+   :lines: 59-80
+   :caption: ASAP7 default power straps setting
+..
+
+The default keys for all hammer configs are defined in https://github.com/ucb-bar/hammer/blob/master/hammer/config/defaults.yml, which contains detailed comments on what each key does. Here is the default setting and parameter descriptions for power strap generation.
+
+.. literalinclude:: ../../hammer/config/defaults.yml
+   :language: yaml
+   :linenos:
+   :lines: 560-598
```

Comment:
These are the wrong lines. It should be 570-604: https://github.com/ucb-bar/hammer/blob/4f25d35bc2e601ef21d18e192c084897b3eb6cd1/hammer/config/defaults.yml#L570-L604